### DISCUSSION TITLE
Making layers visible/invisible

### DISCUSSION THREAD

**Discussion by deeplook** [developmentseed/lonboard#639] [Q&A]
I want to load a number of layers and show only some of them depending on some widget settings. With ipyleaflet I used to add and remove layers programmatically, but with lonboard I haven't found and easy way to do this yet. If there is any I'd love to know. And then I thought maybe it would be possible to make entire layers visible and invisible with a single attribute? Any ideas?

**Comment by kylebarron** [comment]
You can set `layer.visible = False` https://developmentseed.org/lonboard/latest/api/layers/base-layer/#lonboard.BaseLayer.visible
